Bdd/features/environment.py

```
@@ -40,6 +43,6 @@ def after_scenario(context, scenario):
                 sock.sendall(b"RELOAD\n")
                 sock.recv(1024)
             time.sleep(0.5)
-        except Exception:
-            pass
+        except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)